config/environments/development.rb

```diff
@@ -7,14 +7,14 @@
   config.cache_classes = false
 
   # :file_store is used by default when no cache store is specifically configured.
-  # config.cache_store = :file_store
+  config.cache_store = :dalli_store
```

Comment:
watch out. I'm changing that too in https://github.com/openfoodfoundation/openfoodnetwork/pull/5146. For the sake of the mobile changes being delivered ASAP, I would open a separate PR for these settings if you think we should discuss about them

Reply:
as for the style changes themselves, I'm pretty happy about them :+1:

Reply:
Oooops! That wasn't supposed to be in there at all...

Reply:
yes, I figured.